lls/cache.c

```diff
@@ -357,33 +356,57 @@ xmit_icmp_ping_reply(struct gatekeeper_if *iface, struct rte_mbuf *pkt)
 	 * According to RFC 792 page 13: to form an echo reply
 	 * message, the source and destination addresses are simply reversed,
 	 * the type code changed to 0, and the checksum recomputed.
+	 *
+	 * According to RFC 3021: the 255.255.255.255 IP broadcast address
+	 * MUST be used for broadcast Address Mask Replies in point-to-point
+	 * links with 31-bit subnet masks. In other cases, the form
+	 * {<Network-number>, -1} should be used.
```

Comment:
This comment should be broken down into the `if` statement below. The first part goes to the body of `if (icmph->icmp_type == ICMP_ECHO_REQUEST_TYPE) {`, and the second part to the body of `} else if (icmph->icmp_type == ICMP_MASK_REQUEST_TYPE) {`.